### ISSUE TITLE
no sound 

### ISSUE THREAD

**Issue by oz123** [gcompris/GCompris-qt#187]
I have successfully installed gcompris-qt on arch linux arm.

I am not working with PulseAudio. 
I have sound working on youtube and with VLC.

I start `gcompris-qt -s` with sound explicitly enabled, but I have no sounds.  When it starts
I hear a few clicks, but after that there is no sound. Here is the log message:
```

[oznt@klein ~]$ gcompris-qt -s

(process:1711): Gtk-WARNING **: Locale not supported by C library.
	Using the fallback 'C' locale.

** (gcompris-qt:1711): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-2WWp1unRFn: Connection refused
Unable to load translation for locale  "en_US" , use en_US by default
exeCount set to:  27
notifyAudioEffects:  true
notifyAudioVoices:  true
qml: enter main.qml (run #27, ratio=1.5384615384615385, fontRatio=1.6216216216216217, dpi=150, sharedWritablePath=/home/oznt/.local/share/GCompris)
propsReply "Method \"GetAll\" with signature \"s\" on interface \"org.freedesktop.DBus.Properties\" doesn't exist\n"
nmReply "Method \"GetDevices\" with signature \"\" on interface \"org.freedesktop.NetworkManager\" doesn't exist\n"
"Object path cannot be empty"
Downloading resource file "data2/voices-ogg/voices-en.rcc"
shared memfd open() failed: Function not implemented
Local resource is up-to-date: "voices-en.rcc"
Successfully registered resource "/home/oznt/.cache/KDE/gcompris-qt/data2/voices-ogg/voices-en.rcc"
Error: "Could not get/set settings from/on resource."
shared memfd open() failed: Function not implemented
Error: "Could not get/set settings from/on resource."
"Unexpected reply signature: got \"\", expected \"a{sv}\""
DownloadManager: shutting down, 0 active jobs
DownloadManager: shutting down, 0 active jobs

```

Can somehere give me an advice?

**Comment by petitlapin** [comment]
Hi,

how did you install it? Via a package manager or did you compile it?
The sounds seem well activated (notifyAudioEffects:  true).
I'm not sure which error is relevant regarding the sound:
> shared memfd open() failed: Function not implemented
I found https://github.com/kunquat/kunquat/issues/580, maybe the kernel is not recent enough

do you have gstreamer packages installed (plugins)?
On my archlinux, I have (pacman -Q | grep -i gstreamer):
gstreamer 1.10.4-1
gstreamer0.10 0.10.36-4
gstreamer0.10-bad 0.10.23-20
gstreamer0.10-bad-plugins 0.10.23-20
gstreamer0.10-base 0.10.36-3
gstreamer0.10-base-plugins 0.10.36-3
gstreamer0.10-ffmpeg 0.10.13-2
gstreamer0.10-good 0.10.31-11
gstreamer0.10-good-plugins 0.10.31-11


Johnny

**Comment by petitlapin** [comment]
Hi,

Have you tried to install the packages and do you still have the issue? Else, I'll close it in a few days if there is no reproducible way to see the issue.

Johnny

**Comment by oz123** [comment]
Hi, I compiled the package from source. It was on arch linux for arm, which I abandoned already.

**Comment by petitlapin** [comment]
OK, then let's close it. If someone else has the issue, we'll open a new bug.
Thank you!

**Comment by Donno191** [comment]
Same issue Raspberry Pi 4B (Raspberry OS)

pi@raspberrypi:~ $ gcompris-qt -s
qt5ct: using qt5ct plugin
exeCount set to:  55
notifyAudioEffects:  true
notifyAudioVoices:  true
PulseAudioService: pa_context_connect() failed
PulseAudioService: pa_context_connect() failed
qml: enter main.qml (run #55, ratio=2.076923076923077, fontRatio=2.2470433639947434, dpi=102, userDataPath=/home/pi/.local/share/GCompris)
Downloading resource file "data2/voices-ogg/voices-en.rcc"
Local resource is up-to-date: "voices-en.rcc"
Successfully registered resource "/home/pi/.cache/KDE/gcompris-qt/data2/voices-ogg/voices-en.rcc"
load configuration for: "smallnumbers"
Downloading resource file "data2/voices-ogg/voices-en.rcc"
qml: Wordlist: loaded 3 levels from file qrc:/gcompris/src/activities/smallnumbers/resource/default-en.json
Local resource is up-to-date: "voices-en.rcc"
PulseAudioService: pa_context_connect() failed
PulseAudioService: pa_context_connect() failed
PulseAudioService: pa_context_connect() failed
PulseAudioService: pa_context_connect() failed
DownloadManager: shutting down, 0 active jobs
DownloadManager: shutting down, 0 active jobs

**Comment by petitlapin** [comment]
> PulseAudioService: pa_context_connect() failed

Hi,

It seems pulseaudio is either not installed or running. If I remember correctly, it should be `pulseaudio -D` to start it

**Comment by Donno191** [comment]
I installed pulse audio the error messages goes away but there is still other things wrong in gcompris. I will just re-install raspberry os with the des 2 img

**Comment by DarkElvenAngel** [comment]
I'm having an issue as well with sound the problem is a bit different from these others I get sound but it doesn't seem to respect system setting as to where to play the sound.  

I downloaded the self installer from the web site I'm running it on a Pi 3 the sound seems to want to play only over HDMI out however I only have a DVI screen.

This problem also exists in the version found in the Raspberry Pi OS Repo ver 0.95-1

No errors are generated as the sound is playing all other software plays sound to the correct device.

for example on the 0.95-1 version install via the repo's some sound effects do play over the headphone jack however the spoken instructions are not I've got this install on two Raspberries and on one I have an HDMI screen and the spoken instructions play on that while sound effects play over the headphone to my external speakers (All sound should be playing of the External Speakers only,)  HDMI Audio is disable on both Raspperries.

If I need to start a new issue I can do that, thank you for your time and this wonder software my 3 year old loves it!
